packages/terra-site/src/Index.jsx

```diff
@@ -49,7 +50,7 @@ import ToggleExamples from './examples/toggle/Index';
 import ToggleButtonExamples from './examples/toggle-button/Index';
 
 // Test Routes
-/* eslint-disable import/first */
+/* eslint-disable import/first, import/no-extraneous-dependencies */
```

Comment:
Remove `import/no-extraneous-dependencies`

Reply:
After discussing with @bjankord we decided to make it a dev dependency for now

Reply:
I believe we need to list components which have not been released as devDependencies instead of dependencies to they are installed correctly. Listing these components as devDependencies raises a lint error though, so we disable the `import/no-extraneous-dependencies` here. Once horizontal rule is released, we can move it to dependencies and then remove turn this lint rule back on.